tox.ini

```diff
@@ -72,10 +73,28 @@ deps =
     -r{toxinidir}/requirements/openvino.txt
 commands =
     coverage erase
-    coverage run --include=anomalib/* -m pytest -ra
-    coverage report -m --fail-under=90
+    coverage run --include=anomalib/* -m pytest tests/pre_merge/ -ra
+    coverage report -m --fail-under=85
```

Comment:
Should we create a new issue to increase the coverage and link it here? Or is it ok for pre-merge to have 85% coverage?

Reply:
https://github.com/openvinotoolkit/anomalib/issues/94